### DISCUSSION TITLE
Welcome to paperless_app Discussions!

### DISCUSSION THREAD

**Discussion by bauerj** [bauerj/paperless_app#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rknightion** [comment]
Hey!
Just stumbled across this repo, as I've recently fallen in love with paperless-NG myself :).

I'm currently trying to get it to build on my Mac for iOS. I saw the note in the readme, and I'd be happy to distribute the app on the iOS app store (if I can get the builds working).

**Reply by bauerj** [reply]
Hey Rob!

That's great to hear and it would be awesome to have the app available for iOS. I think @qcasey is also looking into this so you might want to talk to him first. I'm not sure if he started any work yet.

**Reply by qcasey** [reply]
That is great! It's on my list to do, but I have not started. I can help if needed, just ping me 😃
